src/services/rfqm_service.ts

```diff
@@ -481,6 +488,13 @@ export class RfqmService {
             logger.error({ error }, 'Current gas price is unable to be fetched, marking worker as not ready.');
             return false;
         }
+
+        // check for outstanding jobs from the worker and resolve them
```

Comment:
@phil-ociraptor is the `isWorkerReady` function run before each time the worker consumes from the queue, or just on startup?